Makefile

```diff
@@ -0,0 +1,38 @@
+.SILENT:
+
+CONTRACT_ABI_FILES = \
+	./out/Dispatcher.sol/Dispatcher.json \
+	./out/Mars.sol/Mars.json \
+	./out/Earth.sol/Earth.json
+
+TMP_ABI_DIR = .tmp_abi_vibc
+
+.PHONY: build-contracts abigen clean extract_artifacts
+
+build-contracts:
+	echo "Building contracts"; \
+	forge install; \
+	forge build
```

Comment:
If any of these commands fail the step will fail and cause the action to fail anyway. It would need to be re-ran manually as these are needed for the following steps